components/brave_federated/operational_patterns.h

```diff
@@ -9,9 +9,9 @@
 // users engage with the browser over a collection period. A collection period
 // is divided into collection slots (i.e. 30m intervals). Two timers are
 // instatiated at startup:
-// 1. |collection_slot_periodic_timer_| fires every |collection_slot_size_|/2
+// 1. |collection_timer_| fires every |collection_slot_size_|/2
```

Comment:
we should update the description of `collection_timer_` here. It now fires according to the new Griffin argument